flyteadmin/auth/authzserver/claims_verifier_test.go

```diff
@@ -78,6 +78,7 @@ func Test_verifyClaims(t *testing.T) {
 	t.Run("String scope", func(t *testing.T) {
 		identityCtx, err := verifyClaims(sets.NewString("https://myserver", "https://myserver2"),
 			map[string]interface{}{
+				"client_id": "my-client",
 				"aud": []string{"https://myserver"},
 				"scp": "all",
```

Comment:
Maybe use something other than the default `all` scope so its clear that it was not injected.

Reply:
Hi, so I removed `client_id` and changed `scp` to something else.